public/app/plugins/panel/xychart/TooltipView.tsx

```diff
@@ -101,8 +106,8 @@ export const TooltipView = ({
         </tr>
         <tbody>
           <tr>
-            <th>{xField.name}</th>
-            <td>{fmt(xField, xField.values.get(rowIndex))}</td>
+            <th>{getFieldDisplayName(xField, frame)}</th>
+            <td>{fmt(frame.fields[0], xField.values.get(rowIndex))}</td>
```

Comment:
not sure i understand this change. why `frame.fields[0]` instead of `xField`? :thinking:

Reply:
Reverting these changes back to original results in following

<img width="404" alt="Screenshot 2023-03-27 at 9 26 43 AM" src="https://user-images.githubusercontent.com/22381771/228004970-75c7c572-86aa-4e01-8f50-37801fe8de23.png">

With current changes it seems we lose % context for the x axis

<img width="445" alt="Screenshot 2023-03-27 at 9 27 05 AM" src="https://user-images.githubusercontent.com/22381771/228004999-af1bd1e2-06e1-476a-a9f0-f364dd2faa9f.png">


Given this I'm not sure on why we need this change as well

Reply:
Yes, that looks like a mistake. Clearly it should be `xField`. I will modify and re-check that everything is ok